lib/rubygems/gemcutter_utilities.rb

```diff
@@ -112,6 +126,28 @@ def get_otp
     ask 'Code: '
   end
 
+  def update_scope
+    sign_in_host        = self.host
+    pretty_host         = pretty_host(sign_in_host)
+    update_scope_params = { self.scope => true }
```

Comment:
>  I believe that would remove the need keep the scope on an instance variable per command, and would make things more explicit and easier to read

`scope` is used not only in `sign_in` but also to update scope in `rubygems_api_request`. scope would have to be passed twice in each action, which didn't seem as pleasant to me.
it would definitely make thinks more explicit. let me know if you think we should change it to explicit arg.